common/network-common/src/main/java/org/apache/spark/network/TransportContext.java

```diff
@@ -124,7 +124,7 @@ public TransportContext(
     this.closeIdleConnections = closeIdleConnections;
 
     if (conf.getModuleName() != null &&
-        conf.getModuleName().equalsIgnoreCase("shuffle") &&
+        "shuffle".equalsIgnoreCase(conf.getModuleName()) &&
```

Comment:
hmm... if `conf.getModuleName() != null` is true,  I think NPE won't happen here

Reply:
remove not null logic